src/data.rs

```diff
@@ -1,35 +1,108 @@
 //! Application state.
 
+use std::cell::RefCell;
+use std::collections::HashMap;
 use std::path::PathBuf;
 use std::rc::Rc;
 
+use druid::kurbo::{Affine, BezPath, Point};
 use druid::Data;
+use norad::glyph::{AffineTransform, Contour, ContourPoint, Glyph, PointType};
 use norad::{FontInfo, FormatVersion, MetaInfo, Ufo};
 
 #[derive(Clone, Default)]
 pub struct AppState {
     pub file: FontObject,
 }
 
+type BezCache = Rc<RefCell<HashMap<String, Rc<BezPath>>>>;
```

Comment:
I would find it useful to have a comment (or perhaps type alias) about what `String` represents.